hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -588,6 +571,56 @@
         } // f: mailbox
       ]
     } // R: INTERCEPT_EN
+    { name: "ADDR_MODE"
+      desc: '''Flash address mode configuration
+
+        This register shows the current address mode and pending changes.
+        It is updated by the HW when the command phase completes.
+        '''
+      swaccess: "ro"
+      hwaccess: "hwo"
+      hwext: true
+      tags: [
+        // SW is not able to read back the ADDR_MODE register right after write.
+        // The read back value is committed value, which needs at least a SPI
+        // transaction.
+        // So excluded from CSR automation test.
+        "excl:CsrNonInitTests:CsrExclWrite"
+      ]
+      fields: [
+        { bits: "0"
+          name: "addr_4b_en"
+          swaccess: "rw"
+          hwaccess: "hrw" // Updated by EN4B/EX4B
+          hwqe: true
+          desc: '''4B Address Mode enable.
+
+            This field configures the internal module to receive 32 bits of the
+            SPI commands. The affected commands are the SPI read commands
+            except QPI, and program commands. It is expected for SW to
+            configure this field at the configuration stage and leave the
+            updation to HW until next reset.
```

Comment:
is `updation` a real word :)?

Reply:
lol, that's a new one for me too! Must've had an interesting copy-paste error there.

Reply:
Oh, I see, this actually was in the original.